azkaban-common/src/main/java/azkaban/executor/ExecutionOptions.java

```diff
@@ -298,6 +303,10 @@ public void setSlaOptions(final List<SlaOption> slaOptions) {
     this.slaOptions = slaOptions;
   }
 
+  public boolean isExecutionRetried() { return isExecutionRetried; }
```

Comment:
Please use keyword this to address member variables. Applies to other places.

Reply:
changed as requested.